src/platform/symbols.map

```diff
@@ -1,4 +1,4 @@
-MIR_PLATFORM_2.17 {
+MIR_PLATFORM_2.18 {
```

Comment:
Would this be a major version bump since we're breaking ABI :thinking:

Reply:
No